test/lan_udev_auto_test.rb

```diff
@@ -105,10 +105,29 @@
       .to receive(:Write)
       .with(path(".udev_persistent.rules"), ay_rules)
     allow(Yast::SCR)
-        .to receive(:Write)
-        .and_return 0
+      .to receive(:Write)
+      .and_return 0
 
-      LanUdevAuto.Import({ "net-udev" => udev_rules })
-      LanUdevAuto.Write
-    end
+    LanUdevAuto.Import({ "net-udev" => udev_rules })
+    LanUdevAuto.Write
+  end
+end
+
+describe "LanUdevAuto#Import" do
+  it "imports defined udev rules correctly" do
```

Comment:
This does not test that there are imported correctly. It only tests it does not crash.